### PR TITLE
Add exc_info to logging.

### PR DESCRIPTION
**Pull request by Djabx**
Some traces are missing in slo-generator

@ocervell

### CODE REVIEW

**Comment by ocervell** [comment]
LGTM, please increase the bugfix version number in `setup.py` as well before we merge. Thanks !

**Comment by jaketf** [comment]
CI is sad please run `make fmt` at root of repo.

```
Step #0 - "lint": ./slo_generator/utils.py:1:1: E902 TokenError: EOF in multi-line statement
Step #0 - "lint": Some files need to be formatted in ./tools/slo-generator - FAIL
```

**Comment by Djabx** [comment]
- [x]  `fmt`
- [x]  rebase on `master`

File: tools/slo-generator/slo_generator/utils.py [hunk 1]
```diff
@@ -187,9 +190,7 @@ def import_dynamic(package, name, prefix="class"):
     except Exception as exception:  # pylint: disable=W0703
         LOGGER.error(
             f'{prefix.capitalize()} "{package}.{name}" not found, check '
-            f'package and class name are valid, or that importing it doesn\'t '
-            f'result in an exception.')
```
Comment (ocervell) [inline tools/slo-generator/slo_generator/utils.py:191]: Lint + running into this fails, can you please restore those two lines ?

Comment (Djabx) [reply tools/slo-generator/slo_generator/utils.py:191]: Sorry about that...

**Comment by ocervell** [comment]
LGTM. @jaketf could you please merge ? Thanks !

**Comment by ocervell** [comment]
@AdrienWalkowiak could you please merge this ? Thank you 👍

**Comment by ocervell** [comment]
@Djabx , could you open this PR in the new repo https://github.com/google/slo-generator ?
You can checkout your branch and run:
```
git subtree split -P tools/slo-generator -b <NEW_BRANCH>
```
and open a PR from <NEW_BRANCH> to the new repo.

**Comment by Djabx** [comment]
> @Djabx , could you open this PR in the new repo https://github.com/google/slo-generator ?
> You can checkout your branch and run:

On my way.

**Comment by ocervell** [comment]
@Djabx yes please close this one.

**Pull request closed without merging**